buf.gen.openapiv2.yaml

```diff
@@ -0,0 +1,4 @@
+version: v1
+plugins:
+  - remote: buf.build/protocolbuffers/plugins/python:v21.9.0-1
+    out: ../gen/
```

Comment:
@eapolinario you can delete the entire repo, You just need `buf generate buf.build/grpc-ecosystem/grpc-gateway --output gen ` against your gen file 🙌